### ISSUE TITLE
Disable accessibility on `playground-browser`

### ISSUE THREAD

**Issue by distantnative** [getkirby/getkirby.com#1289]
Somehow the `aria-hidden`/`role=presentation` got lost again, which is why it's throwing an accessibility warning for this element (while it should actually be ignored).

**Comment by bastianallgeier** [comment]
It should have a role=img

**Comment by distantnative** [comment]
Seems like this isn't sufficient:
<img width="757" alt="Screen Shot 2021-04-16 at 18 45 03" src="https://user-images.githubusercontent.com/3788865/115057066-e152d900-9ee3-11eb-8e74-e4954c90b009.png">

**Comment by bastianallgeier** [comment]
Maybe we can ask @sebastiangreger about his opinion. I don't see how we could improve this any more. The headline is meant to be a part of a visual and not to be read out loud. We already mark it properly as far as I could find out in the HTML. I just don't know how to ignore the contrast issue. If this was an actual img with the same contrast issues, lighthouse would accept it.

**Comment by sebastiangreger** [comment]
Always happy to nerd out over some delicious semantics :wink:

I don't have experience with this particular context of a figure consisting of HTML elements, but my hunch is that `role="img"` is the culprit here.

Lighthouse checking all elements within the tag's content actually makes sense. Displaying an image with essential information where the text does not meet the contrast criteria would be a violation of [WCAG criterion 1.4.11](https://www.w3.org/TR/WCAG21/#non-text-contrast). It's just that automated testing can't evaluate that for bitmaps (so e.g. for an important infographic PNG with poor contrast, LH might award a 100% score while a human reviewer would flag a WCAG violation).

![a11yexplorer](https://user-images.githubusercontent.com/6355217/115397616-963c0d00-a1e6-11eb-9405-c108c0015aca.png)

So `role="img"` does not make LH skip the element's content. Since there are no other indicators for assistive tech to ignore this "graphic" (as it's represented in the accessibility tree), it gets evaluated ...and subsequently fails the test. We cannot really add `aria-hidden="true"` or `role="presentation"` to the `<figure>` element itself, as that would misrepresent the content to an AT user (we would not have an alternative representation of non-text content; [WCAG 1.1.1](https://www.w3.org/TR/WCAG21/#non-text-content) failure).

Since this particular graphic is illustrational to the degree that reading the text in it is not "required for understanding" (quote WCAG re when contrast requirements apply for images), we'd need to communicate that to the assistive tech, so LH can ignore it.

As semantically turning the `<figure>` tag into an `img` using ARIA role appears to cause its content to be validated, it might be better to retain its native role of `figure` (with the `aria-label` correctly describing the illustration as is) and deal with its content separately. This probably requires an extra container element around the illustration's HTML code, which could then be marked with `role="presentation"` or `aria-hidden="true"` (or apply those to all child elements of the figure, but that seems excessive).

```
<figure role="img" aria-label="A browser window with an example website" class="playground-medium-browser bg-white shadow-2xl rounded-xl overflow-hidden">
    <div aria-hidden="true">
        <header class="p-3 flex">
        ...
```

Whether this is truly accessible with AT and passes the LH check as well would have to be evaluated. Most importantly, I am not 100% sure is it allowed to have a `figure` element that does not contain any element accessible with AT.

**Comment by bastianallgeier** [comment]
Thanks for all your input @sebastiangreger. I will try the wrapper first.

**Comment by bastianallgeier** [comment]
Ok, it turns out that Lighthouse doesn't make any difference here. I wonder if we should actually try to make Lighthouse happy in this case.

**Comment by sebastiangreger** [comment]
> Ok, it turns out that Lighthouse doesn't make any difference here. I wonder if we should actually try to make Lighthouse happy in this case.

Hmm. Did you try both `role="presentation"` and `aria-hidden="true"`. I just realized I can try this in the Devtools Inspector. At least in Firefox, the latter effectively removes the wrapper div and all its contents from the Accessibility tree, while the "presentation" role does not (to be understood, why...)

**Comment by sebastiangreger** [comment]
Ah, here it is: https://www.w3.org/TR/using-aria/#presentation 

> For elements with no required children, any elements nested inside the element with role=presentation/none preserve their semantics.

...so, indeed `aria-hidden="true"` would be the only valid option for the wrapper as the `role` attribute only extens to required children (like `<td>` elements inside a `<table role="presentation">`).

There's always something new to learn...

**Comment by bastianallgeier** [comment]
Ahh, ok. I only tried the role.

**Comment by distantnative** [comment]
@bastianallgeier Did you ever try it with `aria-hidden="true"`?

**Comment by bastianallgeier** [comment]
@distantnative yes, I think I did. But I would need to test again to verify.
